include/ultra64/rdp.h

```diff
@@ -0,0 +1,52 @@
+#ifndef ULTRA64_RDP_H
+#define ULTRA64_RDP_H
+
+/* DP Command Registers */
+
+#define DPC_REG_BASE 0xA4100000
```

Comment:
These should all use their physical addresses, i.e.
```suggestion
#define DPC_REG_BASE 0x04100000
```
`HW_REG` will OR these with 0xA0000000 to form the uncached kseg0 address